en/docs/guides/login/logging-into-simplesamlphp-using-is.md

```diff
@@ -0,0 +1,209 @@
+# Logging in to SimpleSAMLphp using WSO2 Identity Server
+
+This section explains how to configure the WSO2 Identity Server with
+SimpleSAMLphp as a service provider. Initially, it is necessary to setup
+SimpleSAMLphp as a service provider. The steps below are tested with
+Ubuntu.
+
+## Setup SimpleSAMLphp as a service provider
+
+1.  Install Apache.
+
+    ``` java
+    # apt-get install apache2 
+    ```
+
+2.  Install PHP and related extensions.
+
+    ``` java
+    # apt-get install php5  
+        
+    # apt-get install php5-cli  
+        
+    # apt-get install php5-common  
+        
+    # apt-get install php5-curl  
+        
+    # apt-get install php-pear  
+        
+    # apt-get install php5-mcrypt 
+    ```
+
+    !!! info 
+		For Ubuntu users, please install the following extension as well:
+		`            # apt-get install php5-json           `
+
+3.  Install SimpleSAMLphp using the following commands.
+
+    ``` java
+    # sudo mkdir /var/simplesamlphp/
+
+    # cd /var/simplesamlphp/  
+        
+    # wget https://github.com/simplesamlphp/simplesamlphp/releases/download/simplesamlphp-1.11.0/simplesamlphp-1.11.0.tar.gz  
+        
+    # tar xvf simplesamlphp-1.11.0.tar.gz  
+        
+    # mv simplesamlphp-1.11.0 simplesamlphp  
+        
+    # cd simplesamlphp  
+        
+    # cp -r metadata-templates/*.php metadata/  
+        
+    # cp -r config-templates/*.php config 
+    ```
+
+4.  Configure SimpleSAMLphp web in Apache.
+
+    ``` java
+    # cd /var/www/html
+        
+    # ln -s /var/simplesamlphp/simplesamlphp/www simplesaml 
+    ```
+
+5.  Start Apache.
+
+    ``` java
+    # apachectl start  
+    ```
+
+6.  Access the SimpleSAMLphp web app from the following location:
+    [http://localhost/simplesaml](http://localhost/simplesaml).
+7.  Set the SimpleSAMLphp administrator login configuration as follows:
+
+    ``` java
+    # cd /var/simplesamlphp/simplesamlphp  
+        
+    # vi config/config.php  
+    ```
+
+    1.  Search for ' **auth.adminpassword** ' and change its value
+        from the default and save the file.
```

Comment:
which file? is it config.php?

If so, shall we add a step saying to open the config.php file and where it is found?

Reply:
This needs to be tested. Will add details once it's done.